examples/notebooks/create_deployment.ipynb

```diff
@@ -2,345 +2,283 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
```

Comment:
Are these changes auto-generated? If so, could you split the generated change into a separate commit? That will make it easier to review

Reply:
I actually edited these files in an IDE that has an integration with Jupyter notebook integration (IntelliJ IDEA, fwiw) so it gives a pretty convenient way to edit the file in a convenient way.
Nothing was auto-generated here.
I guess it might've added some additional fields which it wasn't supposed to.
Let me see if I can reduce the amount of changes here.

Reply:
@roycaihw OK I was able to clean up the mess that was made there, it's looking better now.
To sum up: I updated the versions to use the current ones, and removed the examples that are no longer relevant (`Create DeploymentRollback object`). There was also an empty cell at the bottom that didn't seem to have any purpose, so I took the liberty of removing that as well.
Please let me know if any further changes are needed.
Thanks!